spec/unit/network/formats_spec.rb

```diff
@@ -333,14 +333,6 @@ def self.from_binary(data)
       expect(json.weight).to eq(15)
     end
 
-    it "should use a native parser implementation" do
-      expect(JSON.parser.name).to eq("JSON::Ext::Parser")
-    end
-
-    it "should use a native generator implementation" do
-      expect(JSON.generator.name).to eq("JSON::Ext::Generator")
-    end
-
```

Comment:
I'm not sure if there is a good replacement for these tests, now that we might get either MultiJson or the native gem at runtime. I considered adding a method to the wrapper class that could report on what it is using, but I'm not sure if that's worth it.

Reply:
I was mostly just concerned that it wasn't the pure json implementation because things would "work" but performance would tank.

Reply:
So long as we don't supply a different backend agent-side, this test will technically continue to pass. MultiJson will pull in JSON, so we can use it in this test, kind of by coincidence. It will only start failing if we switch out the backend we're using. If that seems okay and useful, I can put these back in.